app/models/opportunity.rb

```diff
@@ -1,5 +1,14 @@
 class Opportunity < ActiveRecord::Base
+  include PgSearch
+  pg_search_scope :search_by_city,
+    against: [:city]
+  pg_search_scope :search_by_cause,
+    associated_against: {
+    cause: [:cause]
+  }
```

Comment:
Indent the right brace the same as the start of the line where the left brace is.